third_party/proton/dialect/lib/TritonProtonToLLVM/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+add_triton_library(TritonProtonToLLVM
+    RecordOpToLLVM.cpp
+
+    DEPENDS
+    TritonGPUConversionPassIncGen
+
+    LINK_LIBS PUBLIC
```

Comment:
Are you sure it requires all these libraries?

Reply:
Updated